fastlane/README.md

```
@@ -0,0 +1,42 @@
+# Installation
+
+Make sure you have the latest version of the Xcode command line tools installed:
+
+```sh
+xcode-select --install
+```
+
+For _fastlane_ installation instructions, see [Installing
+_fastlane_](https://docs.fastlane.tools/#installing-fastlane)
+
+# Available Actions
```

Markdownlint (0.37.0)

Expected: setext; Actual: atx
Heading style

(MD003, heading-style)